tools/cgrep.py

```diff
@@ -1,93 +0,0 @@
-# Copyright 2011 Google Inc.
```

Comment:
I will let this be then. I may move it back or into a new folder during some future reorganization.